python/tests/core/metrics/test_metrics.py

```diff
@@ -155,7 +155,9 @@ def test_frequent_items_handling_int_as_string() -> None:
     df = pd.DataFrame({"int": [1, 1, 1]})
```

Comment:
If you give the frequent item sketch something with a mix of frequent collisions and then also some noise and enough entries the estimate bounds will differ, maybe try this to make sure that the lower != upper estimate for a test that can catch this bug:

```suggestion
    df_gamma = pd.DataFrame({"feature1": np.random.gamma(1, 2, 1000).astype(int)})
    df_rand =  pd.DataFrame({"feature1": np.random.randint(10000, size=9000)})
    df = df_gamma.append(df_rand)
```